### PR TITLE
Update all references of SMPS to SMP

### LINKED ISSUE #80: Change project name from SMPS to SMP
**Current State:**
SMP is shorter and gets people less tripped up. It doesn't need "Specification" in the title. Many people will already add the word _specification_ as a descriptor when describing SMPS, so the "S" often becomes redundant.

**Desired State:**
- [ ] Change this repository's name (drop "specification")
- [ ] Remove the "S" in SMPS in the logo.
- [ ] Update elsewhere....?

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by leecalcote**
**Description**

This PR fixes #80

**Notes for Reviewers**
- This repo has yet to change names, so the old references were untouched.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/collections/_posts/2020-05-21-analyzing-with-smps.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ Such tests are complicated in part, because there are different types of perform
...
+Outside of the different types of performance tests, performance management concerns include the need for performance and overhead data under a permutation of different workloads (applications) and different types and sizes of infrastructure resources. The need for cross-project, apple-to-apple comparisons are also desired in order to facilitate a comparison of behavioral differences between service meshes and which one might be best-suited for your workloads. Individual projects shy from publishing test results of other, competing service meshes. The need for an independent, unbiased, credible standard measurement is needed is why the Service Mesh Performance Specification (SMP) was created.
```
Comment (PluckyPrecious) [inline docs/collections/_posts/2020-05-21-analyzing-with-smps.md]: check out this, there's a "specification" here.

File: docs/README.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
-# Contributing to the Service Mesh Performance Specification Documentation
+# Contributing to the Service Mesh Performance Documentation
```
Comment (PluckyPrecious) [inline docs/README.md:1]: This repo has "specification" too

File: docs/README.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 
-* *The Service-Mesh-Performance-Specification Docs site is built using Jekyll - a simple static site generator! You can learn more about Jekyll and setting up your development environment in the [Jekyll Docs](https://jekyllrb.com/docs/).*
+* *The Service-Mesh-Performance Docs site is built using Jekyll - a simple static site generator! You can learn more about Jekyll and setting up your development environment in the [Jekyll Docs](https://jekyllrb.com/docs/).*
 
 * First [install Ruby](https://jekyllrb.com/docs/installation/), then install Jekyll and Bundler.
 
 ### 2. Get the code
 
-* Fork and then clone the [Service Mesh Performance Specification repository](https://github.com/layer5io/service-mesh-performance-specification)
+* Fork and then clone the [Service Mesh Performance repository](https://github.com/layer5io/service-mesh-performance-specification)
```
Comment (PluckyPrecious) [inline docs/README.md:13]: repo and command line has specification.

File: docs/README.md [hunk 4]
```diff
@@ -41,7 +41,7 @@ Before contributing, please review the [Documentation Contribution Flow](https:/
   ```bash
   $ git push origin <branch-name>
   ```
-* Then on Github, navigate to the [Service-Mesh-Performance-Specification repository](https://github.com/layer5io/service-mesh-performance-specification) and create a pull request from your recently pushed changes!
+* Then on Github, navigate to the [Service-Mesh-Performance repository](https://github.com/layer5io/service-mesh-performance-specification) and create a pull request from your recently pushed changes!
```
Comment (PluckyPrecious) [inline docs/README.md:44]: and this repo too.

File: .github/workflows/slack.yml [hunk 5]
```diff
@@ -13,4 +13,4 @@ jobs:
         SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
       uses: pullreminders/slack-action@master
       with:
-        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"Someone just starred SMPS! (https://github.com/layer5io/service-mesh-performance-specification/stargazers) Total ⭐️: ${{env.STARS}}\"}'
\ No newline at end of file
+        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"Someone just starred SMP! (https://github.com/layer5io/service-mesh-performance-specification/stargazers) Total ⭐️: ${{env.STARS}}\"}'
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply .github/workflows/slack.yml:16]: @PluckyPrecious for now, yes. I left a small note under the "Notes to reviewers" section in the PR description to callout that the rename of the repo is a remaining task. Good eye here.

File: docs/README.md (hunk 3 above)
Comment (leecalcote) [reply docs/README.md:13]: Good eye. This repo has yet to change names, so the old references were untouched.

File: docs/README.md (hunk 4 above)
Comment (leecalcote) [reply docs/README.md:44]: Good eye. This repo has yet to change names, so the old references were untouched.

**Comment by leecalcote** [comment]
@JubayerJoy given all the work you've done, let's give your PR a shot first, before mine.

File: docs/collections/_posts/2020-05-21-analyzing-with-smps.md (hunk 1 above)
Comment (leecalcote) [reply docs/collections/_posts/2020-05-21-analyzing-with-smps.md]: Ooh, you're right.

File: docs/README.md (hunk 2 above)
Comment (leecalcote) [reply docs/README.md:1]: Oh, good call. This one needs to be removed, too.

**Comment by leecalcote** [comment]
Adding the `pr/on hold` label so that we might consider @JubayerJoy's in place of this PR.

**Comment by JubayerJoy** [comment]
@leecalcote, you have covered almost everything I had on my PR, thanks for giving it a shot!
I have submitted a new PR as previous PRs commits were not signed-off.

**Comment by leecalcote** [comment]
> @leecalcote, you have covered almost everything I had on my PR, thanks for giving it a shot!
> I have submitted a new PR as previous PRs commits were not signed-off.

Very good! I consider it more important that we get you engaged and your work recognized. My feeble commits are easily set aside here. 😉

**Comment by leecalcote** [comment]
Good to close this PR thanks to the efforts of @JubayerJoy.

**Pull request closed without merging**